test_schools_ks.py

```diff
@@ -0,0 +1,111 @@
+import csv
+import os
+import subprocess
+import sys
+import xml.etree.ElementTree as ET
+from pathlib import Path
+from pprint import pprint
+from sys import stderr
+
+from utility import position_on_edge
+
+if 'SUMO_HOME' in os.environ:
+    tools = os.path.join(os.environ['SUMO_HOME'], 'tools')
+    sys.path.append(tools)
+else:
+    sys.exit("Please declare environment variable 'SUMO_HOME' to use sumolib")
+
+import sumolib
+
+
+class TestInstance:
+    def __init__(self, name: str, net_file: str, gen_stats_in_file: str, real_stats_file: str):
+        self.name = name
+        self.net_file = net_file
+        self.gen_stats_in_file = gen_stats_in_file
+        self.real_stats_file = real_stats_file
+
+        try:
+            Path(self.net_file).resolve(strict=True)
+            Path(self.gen_stats_in_file).resolve(strict=True)
+            Path(self.real_stats_file).resolve(strict=True)
+        except FileNotFoundError:
+            print(f"Files for test instance: {self.name} does not exist", file=stderr)
+            pprint(self.__dict__)
+            exit(1)
+
+
+test_instances = [
+    TestInstance("aalborg", "in/cities/aalborg.net.xml", "in/cities/aalborg.stat.xml", "stats/aalborg.stat.xml"),
+    TestInstance("esbjerg", "in/cities/esbjerg.net.xml", "in/cities/esbjerg.stat.xml", "stats/esbjerg.stat.xml"),
+    TestInstance("randers", "in/cities/randers.net.xml", "in/cities/randers.stat.xml", "stats/randers.stat.xml"),
+    TestInstance("slagelse", "in/cities/slagelse.net.xml", "in/cities/slagelse.stat.xml", "stats/slagelse.stat.xml"),
+    TestInstance("vejen", "in/cities/vejen.net.xml", "in/cities/vejen.stat.xml", "stats/vejen.stat.xml")
+]
+
+
+def write_school_coords(net: sumolib.net.Net, stats: ET.ElementTree, filename):
+    """
+        Writes all schools' positions found in stats file to a csv called 'filename'
```

Comment:
Maybe add something on why this is useful - e.g. for testing in R/Matlab

Reply:
77bd93e1bfbd15d787b21395832e6874c2f6fc3c